syntax_and_semantics/attributes.md

```diff
@@ -55,7 +55,7 @@ needed or used outside it.
 
 ## Packed
 
-Allows marking a [C struct](c_bindings/struct.html) as packed, which makes the alignment of the struct to be one byte, and that there is no padding between the elements. In non-packed structs, padding between field types is inserted according to the target system.
+Marks a [C struct](c_bindings/struct.html) as packed, which prevents the automatic insertion of padding bytes between fields in order to align the fields. This is typically only needed if the C library explicitly uses packed structs.
```

Comment:
I think it's better left as "prevents the automatic insertion of padding bytes between fields."